### ISSUE TITLE
Increase disk space on UK prod

### ISSUE THREAD

**Issue by RachL** [openfoodfoundation/openfoodnetwork#10107]
## Description

It's a patch before having the time to fix the underlying issue; See https://openfoodnetwork.slack.com/archives/CDLKH9MM0/p1670174312119459

## Acceptance Criteria & Tests

I'm not sure what UK has currently and how much we should add - If someone can complete this issue?

**Comment by lin-d-hop** [comment]
Looking at DO we seem to be on a suboptimal deal.... we're on the grey one:
![Screenshot from 2022-12-07 05-46-11](https://user-images.githubusercontent.com/8669848/206098650-50f88cf2-aa4f-4fa6-8dc3-b037f3c58260.png)

**Comment by lin-d-hop** [comment]
Okay this is done.

**Comment by dacook** [comment]
I just randomly happened to look at the graph and was quite surprised to see this drop:

![Screen Shot 2022-12-09 at 8 34 36 pm](https://user-images.githubusercontent.com/4188088/206671108-67f657c0-0bcf-4f97-a87e-a22230a8409e.png)

This is a graph of the disk usage in gigabytes, not the percentage, so I didn't expect it to go down.
I logged into the server and can see that the postgres write-ahead logs has been cleared. Before it was 35gb, now only 80mb, which accounts for the drop.
My assumption is that the server was restarted for the disk upgrade, and this cause the write-ahead logs to be cleared.

So, there is now _plenty_ of free space on the server :D 

But if the problem still exists, we may continue to see the disk usage grow and become a problem again. We should keep an eye on this, otherwise it could get out of hand while we forget about it. So I've created a new issue: https://github.com/openfoodfoundation/openfoodnetwork/issues/10132
